src/network-config.js

```diff
@@ -24,7 +25,7 @@ export const networkConfigs = {
       type: 'main', // as returned by web3.eth.net.getNetworkType()
     },
     providers: [
-      { id: 'injected' },
+      { id: 'provided' },
```

Comment:
The only string of changes for the new `use-wallet` api with the `provided-connector` would be changing the id's used for activation; quite clean I'd say!